src/mdns/mdns_mdnssd.cpp

```diff
@@ -1057,6 +1065,8 @@ void PublisherMDnsSd::HostSubscription::Resolve(void)
 
     assert(mServiceRef == nullptr);
 
+    mMDnsSd->mHostResolutionBeginTime[mHostName] = Clock::now();
```

Comment:
Can we make sure that mHostResolutionBeginTime values will eventually be removed?

Reply:
The `Timepoint` objects will be removed in the `On(Service|Host)(Resolved|ResolveFailed)` call.